test/esm/babel-runtime-corejs3.mjs

```diff
@@ -0,0 +1,69 @@
+import assert from "assert";
+
+export default {
```

Comment:
I wrote the testcases in such a way that hopefully we can use a codemod to convert the testcases to Jest BDD test styles, after Jest supports esm.